autopush/main.py

```diff
@@ -383,8 +383,6 @@ def connection_main(sysargs=None, use_files=True):
     proto = "wss" if args.ssl_key else "ws"
     factory = WebSocketServerFactory(
         "%s://%s:%s/" % (proto, args.hostname, args.port),
-        debug=args.debug,
-        debugCodePaths=args.debug,
```

Comment:
are these no longer valid or useful? (Don't know if they might be useful again in the future.)